README.md

```diff
@@ -1,14 +1,18 @@
-# background-checks
+# Background Checks Application
 
-background-checks is an example Temporal Application project.
+This project showcases the implementation of a background checks application with Temporal.
+The application is written in the [Go programming language](https://go.dev/), and uses the 
+Temporal [Go SDK](https://github.com/temporalio/sdk-go).
+See the list of available Temporal SDKs [here](https://docs.temporal.io/). 
 
-This project showcases how to implement a Temporal Application within the context of a [Long Running Human Driven Workflow](./docs/project-narrative.md/#what-is-a-long-running-human-driven-workflow) use case.
+The use case for this application is inspired by the blog post 
+["How Temporal simplified Checkr workflows"](https://docs.temporal.io/blog/how-temporal-simplified-checkr-workflows/)
```

Comment:
I actually would prefer not to change this.

The project narrative goes into the inspiration - we want to call out what the use case is directly to align expectations up front.

Reply:
I meant, we should keep this:
```
This project showcases how to implement a Temporal Application within the context of a [Long Running Human Driven Workflow](./docs/project-narrative.md/#what-is-a-long-running-human-driven-workflow) use case.
```